frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -27,8 +35,10 @@ const propTypes = {
   handleSubmit: PropTypes.func.isRequired,
   closeModal: PropTypes.func.isRequired,
   change: PropTypes.func.isRequired,
+  initialValues: PropTypes.objectOf(),
   handleDelete: PropTypes.func.isRequired,
```

Comment:
This should be throwing a warning in the console, `PropTypes.objectOf()` cannot be empty, you could create a customProp for the initialValues and use that instead.

Reply:
done